scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/core/properties/PASchedulerProperties.java

```diff
@@ -82,6 +82,18 @@
      * tolerated failed attempts to ping a node, before the scheduler decides to restart the task running on it */
     SCHEDULER_NODE_PING_ATTEMPTS("pa.scheduler.core.node.ping.attempts", PropertyType.INTEGER, "1"),
 
+    /** On node side, the time in seconds between two attempts to submit the
+     * result of a task to the scheduler. To be used in combination with
+     * {@link PASchedulerProperties#SCHEDULER_NODE_TERMINATE_TASK_ATTEMPTS}
+     */
+    SCHEDULER_NODE_TERMINATE_TASK_FREQUENCY("pa.scheduler.node.terminate.task.frequency", PropertyType.INTEGER, "30"),
```

Comment:
please add the new properties in a different pull request

Reply:
ok